bigchaindb/pipelines/block.py

```diff
@@ -0,0 +1,80 @@
+import rethinkdb as r
+from multipipes import Pipeline, Node
+
+from bigchaindb.pipelines.utils import ChangeFeed
+from bigchaindb import Bigchain
+
+
+class Block:
+
+    def __init__(self):
+        self.bigchain = Bigchain()
+        self.txs = []
+
+    def filter_tx(self, tx):
+        if tx['assignee'] == self.bigchain.me:
+            tx.pop('assignee')
+            return tx
+
+    def delete_tx(self, tx):
+        r.table('backlog')\
+         .get(tx['id'])\
+         .delete(durability='hard')\
+         .run(self.bigchain.conn)
+
+        return tx
+
+    def validate_tx(self, tx):
+        tx = self.bigchain.is_valid_transaction(tx)
+        if tx:
+            return tx
+
+    def create(self, tx, timeout=False):
+        if tx:
+            self.txs.append(tx)
+        if self.txs and (len(self.txs) == 1000 or timeout):
+            block = self.bigchain.create_block(self.txs)
+            self.txs = []
+            return block
+
+    def write(self, block):
+        self.bigchain.write_block(block)
```

Comment:
The default durability in `write_block()` is currently `soft` so it would seem so... unless we change the default durability of `write_block()` to `hard`